pkg/build/api/v1/types.go

```diff
@@ -402,6 +408,17 @@ type SourceBuildStrategy struct {
 	ForcePull bool `json:"forcePull,omitempty"`
 }
 
+// JenkinsPipelineBuildStrategy holds parameters specific to a Jenkins Pipeline build.
+type JenkinsPipelineBuildStrategy struct {
```

Comment:
Haven't we had this discussion before? I thought we agreed this is a Jenkins Pipeline strategy with the definition contained in a Jenkinsfile. Jenkins Pipeline is what this strategy will be known as by Jenkins users, Jenkinsfile is just an implementation detail.